Sources/DeepLearning/Layer.swift

```diff
@@ -367,3 +367,36 @@ public struct Dropout<Scalar>: Layer
         }
     }
 }
+
+@_fixed_layout
+public struct UpSampling2D<Scalar>: Layer
+    where Scalar: BinaryFloatingPoint & Differentiable & TensorFlowScalar {
+
+    @noDerivative public let scale: Int32
+
+    public init(scaleBy: Int32) {
```

Comment:
In initializers, argument labels and arguments do not need to form a phrase. Use `scalar:` instead.

Reply:
Alright. Do you think `size:` would fit better than `scalar:`? (I'm basing this off of https://github.com/keras-team/keras/blob/master/keras/layers/convolutional.py#L1944)

Reply:
Just noticed I had a typo: `scale` not `scalar` :) 

Yes, `size` sounds better.

Reply:
Fixed.